### PR TITLE
VOLDEV-185 Link masthead edit count to Special:Contributions 

### PR DESCRIPTION
**Pull request by ty-a**

### CODE REVIEW

**Comment by mszabo-wikia** [comment]
CR: https://wikia-inc.atlassian.net/browse/SUS-1708

File: extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php [hunk 1]
```diff
@@ -57,23 +57,29 @@
 			<input type="hidden" id="user" value="<?= $user['id']; ?>"/>
 		<? endif; ?>
 		<div class="masthead-info-lower">
-			<div class="tally">
+			<div class="contributions-details tally">
 				<? if ( !empty( $user['registration'] ) ): ?>
 					<? if ( !empty( $user['edits'] ) || ( empty( $user['edits'] ) && !empty( $user['registration'] ) ) ): ?>
-						<em><?= $user['edits'] ?></em>
```
Comment (macbre) [inline extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php]: Do we need to check `!empty( $user['registration'] )` above once again? And doesn't this if statement always evaluate to `true`? :)

**Review by macbre — CHANGES_REQUESTED** [review]

**Review by macbre — APPROVED** [review]

File: extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php (hunk 1 above)
Comment (ty-a) [reply extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php]: Oh wow. I've removed that check.

**Comment by macbre** [comment]
Thanks, @ty-a !

I'm going to merge this PR into a feature branch that we can pass to QA and the we'll merge it to dev.

**Pull request merged**